de/manual/ch_Finance-Quote.xml

```diff
@@ -168,6 +168,13 @@ SYNOPSIS
       Wenn nicht, dann gibt Ihnen dieser Abschnitt eine Anleitung zur Installation.
     </para>
 
+    <note>
```

Comment:
Nicht hier, sondern beim Ziel [~458] würde ich `Abfrage` durch `Aktualisierung` (Abfragen und in die Datei schreiben) ersetzen, also Abfrage nur für `dump` verwenden.